overlay/chord/relay.go

```diff
@@ -52,6 +52,9 @@ func (rr *RelayRouting) GetNodeToRoute(remoteMsg *node.RemoteMessage) (*node.Loc
 		if finger.IsEmpty() {
 			continue
 		}
+		if CompareID(finger.startID, remoteMsg.Msg.DestId) > 0 {
```

Comment:
This is actually not correct, an example:

localNode Id: `0xf0`
finger: `[0xff, 0x01]`
remoteMsg DestId: `0x00`

In this example, next hop should be finger[0]